src/components/menu-item/menu-item.e2e.ts

```diff
@@ -0,0 +1,83 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { html } from "../../../support/formatting";
+import { accessible, focusable, hidden, reflects, renders } from "../../tests/commonTests";
+
+describe("calcite-menu-item", () => {
+  describe("renders", () => {
+    renders("calcite-menu-item", { display: "flex" });
+  });
+
+  it("reflects", async () =>
+    reflects("calcite-menu-item", [
+      {
+        propertyName: "active",
+        value: "true"
+      },
+
+      {
+        propertyName: "target",
+        value: "_blank"
+      }
+    ]));
+
+  describe("honors hidden attribute", () => {
+    hidden("calcite-menu-item");
+  });
+
+  describe("accessible", () => {
+    accessible(html`<calcite-menu> <calcite-menu-item text="calcite"> </calcite-menu-item> </calcite-menu>`);
+  });
+
+  it("is focusable", () => focusable("calcite-menu-item"));
+
+  //todo : debug why calcite-menu-item requires calite-menu as parent for the click to emit event in test.
```

Comment:
This is being caused by the click-handling element (anchor) not stretching along with its host and therefore not being in the center when the click happens (`E2EElement.click()` scrolls and clicks on the center of the element). If you set `inline-size: 100%` on the anchor, it fixes the test, but not sure if that will cause layout issues (design-wise).

Reply:
makes sense. we can mark `inline-size:100%` for anchor element and the design will be similar.